crates/e2e/tests/e2e/ethflow.rs

```diff
@@ -148,122 +131,6 @@ async fn eth_flow_tx_zero_fee(web3: Web3) {
     .await;
 }
 
-async fn eth_flow_tx(web3: Web3) {
```

Comment:
Why is this test removed, it still seems legit?

Reply:
Yeah, I spent some time figuring out why some tests were removed. Would appreciate more context on this.

Reply:
Not actually removed, it's just code diff shows it in a weird way.

Reply:
`local_node_eth_flow_insufficient_fee` and `local_node_eth_flow_zero_fee` are removed.

Reply:
and `local_node_eth_flow_tx` is changed to be actually `local_node_eth_flow_zero_fee` :)